tests/test_wave_energy.py

```diff
@@ -678,34 +678,3 @@ def test_validate_keys_missing_values(self):
         expected_error = (
             ['dem_path', 'wave_base_data_path'], validation.MESSAGES['MISSING_VALUE'])
         self.assertTrue(expected_error in validation_error_list)
-
-    def test_validate_bad_aoi_incorrect_proj_units(self):
-        """WaveEnergy: test validating AOI vector with incorrect units."""
-        from natcap.invest import wave_energy, validation
-

```

Comment:
These tests aren't specific to wave energy, and the same thing is covered in the validation tests. I removed them because the sample data vectors they use have the wrong geometry type, which now breaks.